.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -135,7 +135,6 @@ variables:
   DOTNET_CLI_UI_LANGUAGE: 'en'
   DOTNET_MULTILEVEL_LOOKUP: 0
   NUGET_XMLDOC_MODE: 'skip'
-  DOTNET_GENERATE_ASPNET_CERTIFICATE: false
```

Comment:
I had to remove this because we were getting crashes in the .NET 8 and .NET 9 on Windows grpc tests 😅 

![image](https://github.com/user-attachments/assets/b5ea780e-9db9-415c-aa6e-b44767517df3)